### ISSUE TITLE
error in FFmpeg.php

### ISSUE THREAD

**Issue by jettero777** [mgp25/Instagram-API#3086]
# Bug Report

---

### Notes:
We will close your issue, sometimes without answering, if any of the following are met:
* You have not included your code or your debug log
* You are asking about `challenge_required`, `checkpoint_required`, `feedback_required` or `sentry_block`. They've already been answered in the Wiki and *countless* closed tickets in the past!
* You have used the wrong issue template
* You are posting screenshots, which are too painful to help you with 

Please make sure you have done the following before submitting your issue:
* You have [Searched](https://github.com/mgp25/Instagram-API/search?type=Issues) the bug-tracker for similar issues including **closed** ones
* You have [Read the FAQ](https://github.com/mgp25/Instagram-API/wiki/FAQ)
* You have [Read the wiki](https://github.com/mgp25/Instagram-API/wiki)
* You have [Reviewed the examples](https://github.com/mgp25/Instagram-API/tree/master/examples)
* You have [Installed the api using ``composer``](https://github.com/mgp25/Instagram-API#installation)
* You are [Using latest API release](https://github.com/mgp25/Instagram-API/releases)

---

### Description
Please include a well-worded description of your issue below; We will close your issue if we have to guess what you're talking about, please be as specific as possible:

Hello, unfortunately commit 253c8419272e0702b93311a03e60be7e05afeea1 introduced a bug, this error in FFmpeg.php line 122:
`array_unshift() expects parameter 1 to be array, string given`

Trace:
```
#0 [internal function]: yii\base\ErrorHandler->handleError(2, 'array_unshift()...', '/var/www/...', 122, Array)
#1 /var/www/vendor/mgp25/instagram-php/src/Media/Video/FFmpeg.php(122): array_unshift('-i '/var/www/ht...', '/var/www/ffmpeg', '-v', 'error')
#2 /var/www/vendor/mgp25/instagram-php/src/Media/Video/FFmpeg.php(94): InstagramAPI\Media\Video\FFmpeg->runAsync('-i '/var/www/ht...')
#3 /var/www/vendor/mgp25/instagram-php/src/Request/Internal.php(2510): InstagramAPI\Media\Video\FFmpeg->run('-i '/var/www/ht...')
#4 /var/www/vendor/mgp25/instagram-php/src/Request/Internal.php(2073): InstagramAPI\Request\Internal->_splitVideoIntoSegments(6, Object(InstagramAPI\Media\Video\VideoDetails))
```

**Comment by vinhosdeluxo** [comment]
#3090 pull request

**Comment by Juan-de-Costa-Rica** [comment]
TF can see her 5

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by JRoy** [comment]
Should be fixed in f898d3ea580f8acd6f06decaee017cbfec96231a
